src/docs/doxygen/mainpage.md

```diff
@@ -20,3 +20,7 @@
 * [Solver configuration](@ref solver_config.hpp)
 * [First order ODE solver](@ref serac::mfem_ext::FirstOrderODE)
 * [Second order ODE solver](@ref serac::mfem_ext::SecondOrderODE)
+
+#### Weak Form Interface ####
+
+* [Integral](@ref serac::Integral)
```

Comment:
I think `weak_form` should be added here as well. Also, all of the new `hpp` files need doxygen file annotations so they show up in the file list.

Reply:
It sounds like there's a gross technical reason this is hard, so I'll resolve this comment for now.